airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/ReplicatorWorkflow.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.sync;
+
+import io.airbyte.config.StandardSyncInput;
+import io.airbyte.config.StandardSyncOutput;
+import io.airbyte.scheduler.models.IntegrationLauncherConfig;
+import io.airbyte.scheduler.models.JobRunConfig;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+@WorkflowInterface
+public interface ReplicatorWorkflow {
+
+  @WorkflowMethod
+  StandardSyncOutput run(JobRunConfig jobRunConfig,
+                         IntegrationLauncherConfig sourceLauncherConfig,
+                         IntegrationLauncherConfig destinationLauncherConfig,
+                         StandardSyncInput syncInput);
```

Comment:
We need to encapsulate the arguments in a POJO, this will make adding a new parameter easier as explained in https://docs.temporal.io/docs/java/workflows/#workflow-interface (search for backward)

Reply:
Since this is just an activity that matches the current impl this should be fine.